prime-router/src/main/kotlin/history/db/ReportGraph.kt

```diff
@@ -46,17 +66,13 @@ class ReportGraph(
      *
      */
     fun getMetadataForReports(descendantReportIds: List<UUID>): List<CovidResultMetadata> {
-        val lineage = ancestorLineageCommonTableExpression(descendantReportIds)
-
-        val sourceReportIds =
```

Comment:
Realized that his was not required since the `join` will naturally filter out anything that isn't a source